kie-wb-tests/kie-wb-tests-gui/src/main/java/org/kie/wb/selenium/model/Persp.java

```diff
@@ -51,6 +52,9 @@
     public static final Persp<ProjectAuthoringPerspective> PROJECT_AUTHORING
             = new Persp<>("Authoring", "Project Authoring", ProjectAuthoringPerspective.class);
 
+    public static final Persp<ProjectLibraryPerspective> PROJECT_LIBRARY
+            = new Persp<>("Authoring", "Project Authoring", ProjectLibraryPerspective.class);
```

Comment:
Ah, that confused me a bit :) So just to understand. When I go to Authoring -> Project Authoring I may end up on authoring or library perspective? Depending if I have some project already present?